ios/HeliumWallet.xcodeproj/project.pbxproj

```diff
@@ -192,6 +196,7 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				962862742BDCCAF70073839C /* Mapbox in Frameworks */,
```

Comment:
These mapbox files seem to be added automatically by xcode when I opened the app, couldn't avoid them getting added? Seems to build fine with them though?